app/static/src/app/components/fit/FitTab.vue

```diff
@@ -40,20 +47,48 @@ export default {
 
         const canFitModel = computed(() => store.getters[`${namespace}/${ModelFitGetter.canRunFit}`]);
         const fitModel = () => store.dispatch(`${namespace}/${ModelFitAction.FitModel}`);
+        const cancelFit = () => store.commit(`${namespace}/${ModelFitMutation.SetFitting}`, false);
+
         const iterations = computed(() => store.state.modelFit.iterations);
         const converged = computed(() => store.state.modelFit.converged);
         const fitting = computed(() => store.state.modelFit.fitting);
+        const cancelled = computed(() => iterations.value && !fitting.value && !converged.value);
         const sumOfSquares = computed(() => store.state.modelFit.sumOfSquares);
         const actionRequiredMessage = computed(() => (canFitModel.value ? "" : userMessages.modelFit.cannotFit));
+        const cancelledMsg = computed(() => (cancelled.value ? userMessages.modelFit.cancelled : ""));
+        const iconType = computed(() => {
+            if (cancelled.value) {
+                return "alert-circle";
+            }
+            if (converged.value) {
+                return "check";
+            }
+            return null;
+        });
+        const iconClass = computed(() => {
+            switch (iconType.value) {
+            case "alert-circle":
+                return "text-secondary";
+            case "check":
+                return "text-success";
+            default:
+                return null;
+            }
```

Comment:
Could you use object literal instead? It's up to you, this is based on personal preference.